Makefile

```
@@ -1,38 +1,59 @@
-.PHONY: xiplot install_xiplot build_xiplot deploy run all clean nuke
+.PHONY: install_xiplot setup_build build_xiplot setup_plugins build_test_plugin bundle_plugins build_webdash deploy serve run all clean nuke
 
 all: run
 
 install_xiplot: xiplot/.gitignore
 
 xiplot/.gitignore:
-	git submodule init xiplot
+	git submodule init xiplot && \
 	git submodule update --depth=1 --remote xiplot
 
-build_xiplot: xiplot/.gitignore
+setup_build:
+	rm -rf dist && \
+	mkdir dist
+
+build_xiplot:
 	cd xiplot && \
 	rm -rf dist && \
 	pip install build toml && \
 	pip install -r requirements.txt && \
 	pip install . && \
-	python3 -m build
+	python3 -m build && \
+	python3 ../patches/bundle-dash-app.py && \
+	cp -r xiplot/assets ../dist/ && \
+	cp dist/xiplot-*.*.*-py3-none-any.whl ../dist/ && \
```

Actually, the copying of the .whl needs to happen before the bundling:

```suggestion
	python3 -m build && \
	cp dist/xiplot-*.*.*-py3-none-any.whl ../dist/ && \
	python3 ../patches/bundle-dash-app.py && \
	cp -r xiplot/assets ../dist/ && \
```

---

Thanks for catching that, it's fixed now.